main.go

```diff
@@ -129,6 +140,23 @@ func sendMetrics(event *corev2.Event) error {
 		return err
 	}
 
+	// Add the check status field as a metric if requested. Measurement recorded as the check name.
+	if config.CheckStatusMetric && event.HasCheck() {
+		var tagList = make([]*corev2.MetricTag, 0)
+		var statusMetric = &corev2.MetricPoint{
+			Name:      event.Check.Name + ".status",
+			Value:     float64(event.Check.Status),
+			Timestamp: event.Timestamp,
+			Tags:      tagList,
+		}
+		// bootstrap the event for metrics
+		if !event.HasMetrics() {
+			event.Metrics = new(corev2.Metrics)
+			event.Metrics.Points = make([]*corev2.MetricPoint, 0)
+		}
```

Comment:
This is unreachable. If an event does not `HasMetrics` then the handler will [error on validation](https://github.com/mylesw42/sensu-influxdb-handler/blob/check-status-metric/main.go#L116-L118).
```suggestion
```

Reply:
In my tests this is required. A Normal check doesn't have any metrics, so by using the "-c" flag, we injecting the status result as a metric, which then gets handled in the next for loop. 
The handler now only fails validation if !event.HasMetrics && !event.CheckStatusMetric.